show captures

sagarpatel10101 opened the pull request:
show current number of captures for each player

Commit:
show captures

show current number of captures for each player

Commit:
Merge remote-tracking branch 'origin/ko' into showCaptureScores

fixed conflicts... moved score text views

sshwang commented on app/src/main/java/com/somestupidappproject/betago/utils/ScoringUtils.java:
```diff
@@ -0,0 +1,75 @@
+        } else if (whiteScore > blackScore) {
+            return Stone.WHITE;
+        }
+        return -1;
```
Make new static variable names for player instead of using Stone. Stone color doesn't really work in this context. Same for "return -1"

sagarpatel10101 replied:
yeah I should just return a string of either BLACK, WHITE, or TIE as returning a player doesn't make much sense either (returning both players in the event of a tie seems bad too)

sshwang commented on app/src/main/java/com/somestupidappproject/betago/utils/ScoreCount.java:
```diff
@@ -0,0 +1,44 @@
+        blackScore = initialBlackScore;
+    }
+
+    public int getWhiteScore() {
```
Refactor these getters and setters to take in a color. This will clean up code in ScoringUtils as well

sagarpatel10101 replied:
what was I thinking here.... lol

Commit:
Merge branch 'master' into showCaptureScores

Commit:
updated stuff

add end game activity (not fragment because that was annoying)

added end game trigger logic

implemented very basic scoring
